recipes/libfaketime/test_makefile.patch

```diff
@@ -0,0 +1,8 @@
+--- test/Makefile.orig	2020-01-24 15:55:37.320406491 +0200
++++ test/Makefile	2020-01-24 15:55:52.316279494 +0200
+@@ -1,4 +1,4 @@
+-CC = gcc
++CC ?= gcc
+ 
+ CFLAGS = -std=gnu99 -Wall -DFAKE_STAT -Werror -Wextra $(FAKETIME_COMPILE_CFLAGS)
```

Comment:
```suggestion
 CFLAGS = $(CFLAGS) -std=gnu99 -Wall -DFAKE_STAT -Werror -Wextra $(FAKETIME_COMPILE_CFLAGS)
```

Reply:
This caused a build failure because of a circular variable definition, which is why I went with += instead.